frontend/web/src/contexts/WebSocketContext.tsx

```
@@ -0,0 +1,188 @@
+}) => {
+  const [generationUpdates, setGenerationUpdates] = useState<Map<string, GenerationUpdate>>(
+    new Map()
+  );
+  const [queueStatus, setQueueStatus] = useState<QueueUpdate | null>(null);
+  const [modelStatus, setModelStatus] = useState<Map<string, ModelUpdate>>(new Map());
+  const [systemMessages, setSystemMessages] = useState<SystemMessage[]>([]);
+
+  const handleMessage = (message: WebSocketMessage) => {
+    switch (message.type) {
+      case 'generation_update':
+        const genUpdate = message.payload as GenerationUpdate;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)